src/Components/Form.js

```diff
@@ -44,13 +44,139 @@ class Form extends React.Component {
+        </div>
+        <div>
+          <label htmlFor="recommendation">
+            Would you recommend freeCodeCamp to a friend?
+          </label>
+          <div id="recommendation">
+            <Radio
+              value="definitely"
+              label="Definitely"
+              name="recommendation"
+              checked={true}
+            />
+            <Radio value="maybe" label="Maybe" name="recommendation" />
+            <Radio value="not-sure" label="Not sure" name="recommendation" />
+          </div>
+        </div>
+        <div>
+          <label htmlFor="improved">
+            What would you like to see improved?(check all that apply){" "}
+          </label>
+          <div id="improved">
+            <CheckBox
+              type="checkbox"
+              id="front-end"
+              name="front-end"
+              value="front end projects"
+              label="Front-end Projects"
+            />
+            <CheckBox
+              type="checkbox"
+              id="back-end"
+              name="back-end"
+              value="back end projects"
+              label="Back-end Projects"
+            />
+            <CheckBox
+              type="checkbox"
+              id="data-vis"
+              name="data-vis"
+              value="data visualization"
+              label="Data Visuaization"
+            />
+            <CheckBox
+              type="checkbox"
+              id="challenges"
+              name="challenges"
+              value="challenges"
+              label="Challenges"
+            />
+            <CheckBox
+              type="checkbox"
+              id="open-source"
+              name="open-source"
+              value="open source community"
+              label="Open Source Community"
+            />
+            <CheckBox
+              type="checkbox"
+              id="gitter-help"
+              name="gitter-help"
+              value="gitter help rooms"
+              label="Gitter help rooms"
+            />
+            <CheckBox
+              type="checkbox"
+              id="videos"
+              name="videos"
+              value="videos"
+              label="Videos"
+            />
+            <CheckBox
+              type="checkbox"
+              id="city-meetups"
+              name="city-meetups"
+              value="city-meetups"
+              label="City Meetups"
+            />
+            <CheckBox
+              type="checkbox"
+              id="wiki"
+              name="wiki"
+              value="wiki"
+              label="Wiki"
+            />
+            <CheckBox
+              type="checkbox"
+              id="forum"
+              name="forum"
+              value="forum"
+              label="Forum"
+            />
+            <CheckBox
+              type="checkbox"
+              id="add-courses"
+              name="add-courses"
+              value="add-courses"
+              label="Additional Courses"
+            />
```

Comment:
Let's have a static array contains the data for each checkbox, per example:
[{value: "add-courses",label: "Additional Courses"},{value: "wiki",label: "Wiki"}] 
And start looping over them using .map function